app/assets/javascripts/static_pages.js

```diff
@@ -0,0 +1,16 @@
+//= require medium-editor
+
+WebsiteOne.define('StaticPages', function() {
+    return {
+        init: function() {
+            var editor = new MediumEditor('.editable', {
+                placeholder: {
+                    /* This example includes the default options for placeholder,
+                        if nothing is passed this is what it used */
+                    text: 'Type your text',
+                    hideOnClick: true
+                }
+            }
+        };
+    }
+});
```

Comment:
You could call it later in that same file, although I'm not sure about that define function.

If I recall almost all the controllers seem to use slightly different approaches to JavaScript in this project.

There's also something weird about JavaScript being loaded on only the relevant pages or on all of them. I've needed to add extra calls in the views on occasion.

Does the medium rails gem not specify?